socorro/middleware/search_common.py

```diff
@@ -135,7 +135,7 @@ def build_filters(self, fields):
                 ))
 
                 # Add an interval field.
-                default_interval = 1
+                default_interval = 50
```

Comment:
That's the Elasticsearch default now. This change is a security against the "matrix" problem I had while testing. If we set it to 1, then any histogram over a field that contains big numbers will return a huge number of results and will cause out-of-memory problems. By increasing the default interval we can mitigate that problem and ensure users won't accidentally hit it.